%YAML:1.0
---
image_width:  1280
image_height: 720
camera_matrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [
       790.053572,   0.     , 652.262301,
       0.     , 794.941456, 345.793832,
       0.     ,   0.     ,   1.     ]
dist_coeffs: !!opencv-matrix
   rows: 5
   cols: 1
   dt: d
   data: [ -0.280229, 0.087736, 0.004291, -0.001665, 0.000000 ]

# imagepoints: [
#     643.33, 643.33,
#     486.66, 581.33,
#     612.66, 524.00,
#     760.66, 563.33 ]
# worldpoints: [
#     1.60, 1.60, 00.00,
#     2.40, 1.60, 00.00,
#     2.40, 2.40, 00.00,    
#     1.60, 2.40, 00.00 ]

imagepoints: [
    548.0, 420.0,
    798.0, 418.0,
    856.0, 564.0,
    515.0, 570.0]
worldpoints: [
    6.00, 0.65, 00.00,
    6.00, -1.35, 00.00,
    4.00, -1.35, 00.00,    
    4.00, 0.65, 00.00 ]